_layouts/people.html

```diff
@@ -41,9 +41,9 @@ <h2>{{ group.group }}</h2>
           {% else %}
           {{ person.name }},
           {% endif %}
-          ({{ person.role }}, {{ person.year }}), 
+          ({{ person.role }}, {{ person.year }})
           {% if person.employer_url != null %}
-          <a href="{{ person.employer_url }}" target="_blank">{{ person.employer }}</a>
+          , <a href="{{ person.employer_url }}" target="_blank">{{ person.employer }}</a>
```

Comment:
FYI- with this change, there is no comma in front of the company name if there is no url (see example of the two different behaviors)
![commas](https://user-images.githubusercontent.com/4060017/123272943-d5792980-d4c7-11eb-9766-4b82fd4822e1.png)